SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UnresolvedReferenceGymTest/UnresolvedReferenceGymTestActor.cpp

```diff
@@ -0,0 +1,15 @@
+#include "UnresolvedReferenceGymTestActor.h"
+#include "Net/UnrealNetwork.h"
+
+void AUnresolvedReferenceGymTestActor::GetLifetimeReplicatedProps(TArray<FLifetimeProperty>& OutLifetimeProps) const
+{
+	Super::GetLifetimeReplicatedProps(OutLifetimeProps);
+
+	DOREPLIFETIME(AUnresolvedReferenceGymTestActor, numRepNotify);
```

Comment:
Nit: Unreal has a standard of capitals at the start variables (Pascal case )(except bools)(unreal is weird). Probably a few cases where this can be changed.